### PR TITLE
Task.layout page

### PR DESCRIPTION
**Pull request by punpundada**
New Changes With Layout

### CODE REVIEW

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  const [isLogin, setIsLogin] = useState(false);
+  useEffect(() => {
+    let id = sessionStorage.getItem("userId");
+    id !== null ? setIsLogin(true) : setIsLogin(false);
+  },[setIsLogin]);
+
+  return (
+    <div className="w-screen h-14 bg-gray-900 text-white">
+      <ul className="flex gap-8 p-3 text-2xl ms-6 ">
+        {isLogin ? (
```
Comment (parashkar) [inline src/components/Navbar.tsx:14]: On Logout The NavBar is Not Updating to redirect . The Home Button is Not re rendering to accomodate Changes

**Review by parashkar — CHANGES_REQUESTED** [review]

File: src/components/Navbar.tsx (hunk 1 above)
Comment (punpundada) [reply src/components/Navbar.tsx:14]: Working

**Review by parashkar — APPROVED** [review]
Many Layout Changes

**Comment by parashkar** [comment]
NEW Changes

**Pull request merged**